_posts/pl/2022-01-11-smarthome.md

```diff
@@ -0,0 +1,316 @@
+W kwestii baterii mogę dodać tylko, że ze względu na różnice w napięciu między bateriami a akumulatorami (1.2V vs 1.5V) konieczne może być używanie zwykłych baterii.
+
+![Głowica termostatyczna](/assets/img/posts/2022-01-11-smarthome/trv.png){: width="250" }
+
+#### Czujniki wycieku wody
+
+Konstrukcyjnie dosyć proste urządzenia. Na ich spodzie znajdują się dwie elektrody, których zwarcie powoduje zmianę stanu czujnika.
+W przypadku czujników Xiaomi/Aqara elektrody możemy wykręcić, zmniejszając tym samym poziom, od którego reagują.
+Najstarszy z moich czujników ma już dwa lata, a ciągle raportuje stan baterii na 100%, więc bez obaw można umieszczać je w trudno dostępnych miejscach.
+
+![Czujnik wycieku wody](/assets/img/posts/2022-01-11-smarthome/leaksensor.png){: width="250" }
+
+#### Czujniki temperatury
+
+Jedne z moich pierwszych czujników, a jak wspominałem, zaczynałem z bramką od Xiaomi, stąd wybór też tego producenta.
+Z zalet kwadratowych czujników temperatury Xiaomi mogę wymienić to, że poza temperaturą oraz wilgotnością raportują też ciśnienie atmosferyczne.
+
+![Czujnik temperatury](/assets/img/posts/2022-01-11-smarthome/temperaturesensor.png){: width="250" }
+
+## Bezpieczeństwo i prywatność
+
+W mojej ocenie w całej zabawie w SmartHome warto zwrócić uwagę na prywatność oraz bezpieczeństwo.
+Nie wyobrażam sobie trzymać w sieci urządzeń, które zbierają dane i wysyłają je na Azjatyckie, niesprawdzone serwery.
+W sieci można znaleźć wiele przykładów kamerek, które albo miały słabe zabezpieczenia, albo luki w bezpieczeństwie i pozwalały każdemu na zdalny dostęp.
+Dla mnie podstawowym wymaganiem przy tego typu rozwiązaniach jest praca w sieci lokalnej.
+Wszystkie smart urządzenia mają wycięty dostęp do internetu, a pozwalam na niego jedynie przy okresowych aktualizacjach.
+W przyszłości zastanawiam się nad poluźnieniem tej restrykcji dla wybranych serwerów i urządzeń, ponieważ koliduje mi to z integracją od pralki LG oraz zmywarki Bosch.
+Aktualnie, jeśli mam potrzebę dostępu z zewnątrz to zapewniam sobie ten dostęp sam za pomocą rozwiązań, które uważam za godne zaufania. 
+Przykładem takiego dostępu jest VPN, który sam wystawiam oraz integracja HomeKit od iOS.
+Zdalny dostęp od Apple (przez HomeKit) dostałem tak naprawdę „out of the box". 
+W domu mam zawsze tablet od Apple i tak długo, jak jest naładowany, pozwala innym domownikom z urządzeniami tego samego producenta na zdalny dostęp,
+spinając całość przez chmurę Apple.
+
+## To na tyle...
+
+Zabawa w SmartHome nie należy do najtańszych, ale mam nadzieję, że lektura tego artykułu nieco przybliży temat.
+Do indywidualnej decyzji zostawiam czytelnikowi czy chce zająć się całością sam, czy może z pomocą specjalizujących się w tym firm.
+Jeśli decyzja padnie na firmy, to mam nadzieję, że artykuł nakieruje na możliwe opcje, tak aby nie żałować wydanych pieniędzy.
```

Comment:
Sami musicie zdecydować, czy chcecie zająć się całością sami czy może skorzystać z pomocy specjalistycznych firm.
Jeśli zdecydujecie się na wybór jakieś firmy, to ten artykuł nakieruje Was na możliwe opcje, tak aby nie żałować wydanych pieniędzy.